src/Organizer/OrganizerLDProjector.php

```diff
@@ -402,8 +403,13 @@ function ($label) use ($labelRemoved) {
     private function applyOrganizerDeleted(
```

Comment:
Ook hier is er wat overlap met _Event_ en _Place_. Voor vertaling is er bijvoorbeeld gebruik gemaakt van  een `MultilingualJsonLDProjectorTrait` dat je over kan hergebruiken.